### PR TITLE
Feature/cdhk 191

### PR DESCRIPTION
**Pull request by BenJeau**
## Description

Added colors to the events in Google Calendar

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] If this is a pull request to master, I have updated the version number in package.json
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+// 	green: '#36B478',
+// 	purple: '#7885CA'
+// };
 
-export const calendarEventColorsInside = {
-	red: '#EDA49E',
-	green: '#52CB91',
-	purple: '#9DA6D8'
-};
+// export const calendarEventColorsInside = {
```
Comment (businessglitch) [inline config.js]: Since they are commented out, we can delete that set of code

File: __tests__/schoolschedule-test.js [hunk 2]
```diff
@@ -33,10 +33,10 @@ test("Buttons render correctly", () => {
 	expect(button).toMatchSnapshot();
 });
 
-test('Icons render correctly', () => {
-	const icon = renderer.create(<FontAwesome5 name="university" size={130} color='#ffffff'/>).toJSON();
-	expect(icon).toMatchSnapshot();
-});
+// test('Icons render correctly', () => {
```
Comment (businessglitch) [inline __tests__/schoolschedule-test.js]: Same here

File: __tests__/nonfixedevent-test.js [hunk 3]
```diff
@@ -57,15 +57,15 @@ test('NumericInputs render correctly', () => {
...
-	const slider = renderer.create(<Slider 
-		minimumValue={0}
-		maximumValue={1} 
-		step={0.5}
-		thumbTintColor={'#FF9F1C'}
-		minimumTrackTintColor={'#FFBF69'}/>).toJSON();
-	expect(slider).toMatchSnapshot();
-});
+// test('Slider renders correctly', () => {
+// 	const slider = renderer.create(<Slider
```
Comment (businessglitch) [inline __tests__/nonfixedevent-test.js]: Avoid pushing commented code

**Comment by BenJeau** [comment]
So what, we need to delete every little chunk of commented out code if it could be used later on?

**Pull request merged**